main.go

```diff
@@ -482,6 +638,18 @@ func parse(stats map[string]string, key string) float64 {
 	return v
 }
 
+func parseBool(stats map[string]string, key string) float64 {
+	switch stats[key] {
+	case "yes":
+		return 1
+	case "no":
+		return 0
+	default:
+		log.Errorf("Failed parse %s %q", key, stats[key])
+		return math.NaN()
```

Comment:
I'm not sure if I like this, we prefer to not return `NaN` unless there's a math reason to do so.

In this case I think we should either return 0 for default, or also return an `error` to be checked by the caller.

Reply:
I'm using a `NaN` for an error. it's same usage as `func parse()` , I think.

I will change `func parseBool()` to return an `error` and check by caller.
So `func parse()` should return an `error`. What do you think?

Reply:
Yea, you're right. IMO `func parse()` is also incorrect in this behavior. But for consistency, I think this is fine. We can make a separate issue to fix the behavior separately.